allensdk/brain_observatory/behavior/data_objects/task_parameters.py

```diff
@@ -44,7 +50,8 @@ def __init__(self,
                  stimulus: str,
                  stimulus_distribution: StimulusDistribution,
                  task_type: TaskType,
-                 n_stimulus_frames: int):
+                 n_stimulus_frames: int,
+                 stimulus_name: str = Optional[None]):
```

Comment:
looks like this can never be null  so why is it optional ?

Reply:
When loading from an already created NWB file, this value will not exist hence the default of `None`.